websites/H/Homestuck/presence.ts

```diff
@@ -0,0 +1,310 @@
+const presence = new Presence({
+		clientId: "941798064694378557"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		pathStr = document.location.pathname,
+		pathArr = pathStr.split("/");
+	switch (pathArr[1]) {
+		case "":
+			presenceData.details = "Viewing home page";
+			break;
+
+		case "stories":
+			presenceData.details = "Viewing stories";
+			break;
+
+		case "jailbreak":
+			presenceData.details = "Reading Jailbreak";
+			presenceData.smallImageKey = "mspa";
+			if (!pathArr[2]) presenceData.state = "Page 1 of 134";
```

Comment:
shouldn't you dynamically set the total pages? So you don't have to update the presence if they add a page?

Reply:
i considered doing so, but the only two projects on the site that aren't confirmed to be finished already haven't been touched since pre-2009. figured it'd just be easier to hardcode it